adr/011-dast.md

```diff
@@ -12,15 +15,48 @@ Accepted.
 
 ## Context
 
-Zed Attack Proxy (ZAP) is a free, open-source penetration testing tool being maintained under the umbrella of the Open Web Application Security Project (OWASP). ZAP is designed specifically for testing web applications and is both flexible and extensible.
-- Has Docker image which allows for an isolated testing environment.
-- It is open source and widely used.
-- Provides crossed-platform.
-- Has extensive community support.
-- Can generate reports.
+ZAP is a free, open-source penetration testing tool maintained under the umbrella of the Open Web Application Security Project (OWASP). ZAP is designed specifically for testing web applications and is both flexible and extensible.
 
 [Reference](https://www.zaproxy.org/getting-started/)
 
+## Impact
+
+### Positive
+  
+- Integration Capabilities: ZAP can be integrated into CI/CD pipelines, enabling automated security testing as part of the software development lifecycle.
+
+  
+- User-Friendly Interface: ZAP has an easy-to-use interface that simplifies the process of running scans and analyzing results, making it accessible for teams with varying levels of expertise.
+
+  
+- Comprehensive Reporting: ZAP can generate detailed reports that help identify vulnerabilities and guide remediation efforts, useful for compliance and audits.
+
+
+- Community Support: The active community surrounding ZAP provides a wealth of plugins, resources, and support, facilitating easier troubleshooting and best practices sharing.
```

Comment:
Looks like the bullet points are missing the `**` around them.  Everywhere else seems to have the asterisks.